mkdocs.yml

```diff
@@ -14,5 +14,7 @@ nav:
   - Networking:  networking.md
   - AWS IAM User service:  iam-user-service.md
   - AWS Cost Management Dashboard:  aws-billing-and-cost-management-dashboard-via-quicksight.md
+- FAQs:
```

Comment:
Could this section be more descriptive since we are staying away from FAQs titles. Please provide other suggestions for this category.

Reply:
As per our conversation I updated this to "Upcoming features"